projects/ngneat/error-tailor/src/lib/control-error.directive.spec.ts

```diff
@@ -68,6 +82,20 @@ describe('ControlErrorDirective', () => {
       validator({ controls }: FormArray) {
         return controls.some(control => control.valid) ? null : { requiredone: true };
       }
+
+      asyncValidator({ value }: AbstractControl): Observable<ValidationErrors | null> {
+        return of(value).pipe(
```

Comment:
This isn't async :)

Reply:
Pushed a change.  let me know if you want it resolved with a different approach.